package-lock.json

```diff
@@ -3560,13 +3568,13 @@
         },
         "wrappy": {
           "version": "1.0.2",
-          "resolved": false,
+          "resolved": "https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz",
           "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8=",
           "optional": true
         },
         "yallist": {
           "version": "3.1.1",
-          "resolved": false,
```

Comment:
so odd that this sometimes turns to `false`. I might have something configured wrong. I googled this problem and only found old irrelevant comments about npm bugs. lmk if you know